schema/schema_test.go

```diff
@@ -975,3 +975,288 @@ func TestParseWithUnmatchedDatatype(t *testing.T) {
 		assert.Nil(t, tbl)
 	}
 }
+
+func TestParseSchemaWithSkipFields(t *testing.T) {
```

Comment:
Is it necessary to write such a complicated test case? Can we make it small and clean? It will be easy to maintain.